File: DofusSpa/src/app/_models/product.ts
import { ProductStatistic } from "./product-statistic";
import { User } from "./user";

export interface Product{
    id : number,
    price : number,
    betOn : boolean,
    description : string,
    userID : number,
    status : string,
    quantity : number,
    user : User,
    statistics : ProductStatistic[]
}

File: DofusApi/Models/Product.cs
using System;

namespace DofusApi.Models
{
    public class Product
    {
        public int ID { get; set; }
        
        public double Price { get; set; }
        
        public bool BetOn { get; set; }
        
        public string Description { get; set; }
        
        public int UserID { get; set; }
        
        public string Status { get; set; }
        
        public int Quantity { get; set; }
        
        public User User { get; set; }
        
        
    }
}